### ISSUE TITLE
Patches API Example

### ISSUE THREAD

**Issue by tadscottsmith** [chuot/rdio-scanner#331]
Can you provide an example and update the documentation with an example of the JSON that the /api/call-upload endpoint expects for the "patches"?

**Comment by tadscottsmith** [comment]
It seems the actual format is `[TGID1,TGID2]`. Would probably be good to have the doc updated.

**Comment by chuot** [comment]
You're right, I forgot to change the doc when I added this feature. Anyway, it is indeed [TGID,...] as per Trunk-Recorder is sending them. I'll make sure to fix the doc when I'll publish v7, which I'm getting back to it after some personnal life hickups.

And btw, thanks so much for the SDRTrunk PR, it is very much appreciated!

**Comment by tadscottsmith** [comment]
Sorry for the very delayed response. Thank you for the verification and including this in the upcoming v7 documentation. I'm excited to see the new release!

I'm hopeful that the changes to SDRTrunk will get merged in, but I understand the author doesn't know me from Adam, and I'm sure he's busy as well, so we'll see.

Hope life is treating you better! Take care!
